cmake/CMakeLists.txt

```diff
@@ -410,8 +410,8 @@ else()
 
   # Enabling this for Mac builds causes a sigabrt in several tests.
   if (UNIX AND NOT APPLE)
-    string(APPEND CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -ffunction-sections -fdata-sections")
-    string(APPEND CMAKE_CXX_FLAGS "${CMAKE_C_FLAGS} -ffunction-sections -fdata-sections")
+    string(APPEND CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -ffunction-sections -fdata-sections ")
```

Comment:
Should it be  "string(APPEND CMAKE_CXX_FLAGS " -f...")"   or set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ...")

Reply:
I've seen that both get used.

Reply:
Yes. Both of them should work. But from my point of view, you are doing it in a wrong way. You can continue using APPEND, but don't type CMAKE_CXX_FLAGS twice.